bugbug/commit_features.py

```diff
@@ -137,95 +137,128 @@ class source_code_file_metrics(object):
     name = "metrics on source code file"
 
     def __call__(self, commit, **kwargs):
+        commit_metrics = commit["metrics"]
```

Comment:
This change is unrelated and makes the diff longer. 

Note: If we are going to refactor this, it should be done in a separate PR that is detected for that.

Reply:
okay, thanks. done